Velaptor/UI/Button.cs

```diff
@@ -325,16 +328,23 @@ public override void LoadContent()
                 return;
             }
 
-            Label ??= new Label(this.contentLoader, this.contentLoader.LoadFont(DefaultRegularFont, 12));
+            // If the label is null
+            Label ??= this.controlFactory.CreateLabel(this.cachedText, this.contentLoader.LoadFont(DefaultRegularFont, 12));
 
-            Label.LoadContent();
-            Label.Text = this.cachedText;
+            Label?.LoadContent();
 
             // Update the enabled status of the label to match the button.
             // If the enabled value was set before the content was loaded,
             // it will "cache" the value until the content is loaded
-            Label.Enabled = base.Enabled;
-            Label.Position = Position;
+            if (Label is not null)
+            {
+                Label.Enabled = base.Enabled;
+                Label.Position = Position;
+
+                // Set the text to the cached text value if was set previously
+                // This occurs when the text of the button was set before load content was invoked
```

Comment:
Refactor to multi-line comment and add punctuation accordingly.
```suggestion
                // Set the text to the cached text value if was set previously
                // This occurs when the text of the button was set before load content was invoked
```